src/assets/scripts/client/base/ModelSource/modelSourceClassMap.js

```diff
@@ -17,7 +18,15 @@ export const CLASS_MAP = {
      * @type {FixModel}
      * @final
      */
-    FixModel: FixModel
+    FixModel: FixModel,
```

Comment:
I'm not sure I follow this. Is this in order to do `const thing = new CLASS_MAP.FixModel(data);` as opposed to importing fixModel and doing `const thing = new FixModel(data);`?